packages/video_player/tizen/src/video_player.cc

```diff
@@ -158,7 +158,19 @@ VideoPlayer::VideoPlayer(flutter::PluginRegistrar *plugin_registrar,
   SetUpEventChannel(plugin_registrar->messenger());
 }
 
-VideoPlayer::~VideoPlayer() { Dispose(); }
+VideoPlayer::~VideoPlayer() {
```

Comment:
Is it no longer necessary to call `Dispose` from the destructor? Should a similar thing be applied to video_player_videohole for consistency?

Reply:
```cpp
std::optional<FlutterError> VideoPlayerTizenPlugin::Dispose(
    const TextureMessage &msg) {
  LOG_DEBUG("[VideoPlayerTizenPlugin] textureId: %ld", msg.texture_id());

  auto iter = players_.find(msg.texture_id());
  if (iter != players_.end()) {
    iter->second->Dispose();
    players_.erase(iter);
  }

  return std::nullopt;
}
```
we don't need call dispose int destructor,  will cause call dispose() twice.